x/market/types/state.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by "stringer -linecomment -output=state.go -type=OrderState,BidState,LeaseState"; DO NOT EDIT.
```

Comment:
can you make the file name `autogen_stringer.go` for all of the generated files here?  it makes it easier to quickly know what is auto-generated and what isn't for both us reading it and for doing things like skipping these files during linting.